app/shared/sentry.ts

```diff
@@ -0,0 +1,49 @@
+import * as Sentry from "@sentry/browser";
+import { Extras } from "@sentry/types";
+
+import { BrowserTracing } from "@sentry/tracing";
+
+export const initSentry = () =>
+  Sentry.init({
+    dsn: "https://e4b439185ee94fca9599b768a75d211e@o1298958.ingest.sentry.io/6530661",
```

Comment:
I think you want to get this over env config

Reply:
and make sentry  optional if its not provided?

Reply:
sooooo, I don't have access to env variables here

Reply:
sure, it has to be passed when you call init

Reply:
Yap :(

Reply:
so we need to move it to env, read it probably into config and pass it to initSentry call from a place where we have a config, I guess